detectors/resources/src/test/java/com/google/cloud/opentelemetry/detectors/GCEResourceTest.java

```diff
@@ -0,0 +1,68 @@
+package com.google.cloud.opentelemetry.detectors;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.junit.Assert.*;
```

Comment:
Wildcard imports have a poor readability (hard to see which classes are imported), could you replace it with individual imports please?
FYI you can disable wildcards imports in IntelliJ: https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports